test/fixtures/get-orderbook-response.json

```diff
@@ -324,9 +324,9 @@
           "value": "0.5",
           "counterparty": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B"
         },
-        "totalPrice": {
+        "price": {
```

Comment:
Yeah, that's what I was thinking. `exchangeRate` is always equal to quality, and is computed in javascript for offercreate transactions and account_offers, which only specify takerGets and takerPays.

I'm also thinking maybe orders can be specified by "totalPrice" and "totalPrice" can also be returned in queries as before, but we can add "exchangeRate" to the "properties" section so that users have access to the more robust math.